src/Courses/CourseDetailsTabs/TopicsTab.tsx

```diff
@@ -347,7 +417,7 @@ export const TopicsTab: React.FC<TopicsTabProps> = ({ course, setCourse }) => {
                 headerContent={<h5>Confirm delete</h5>}
                 bodyContent={`Are you sure you want to remove ${confirmationParamters.identifierText}?`}
             />
-            {error && <Alert variant="danger">{error.message}</Alert>}
+            <Alert variant={alert.variant} show={Boolean(alert.message)}>{alert.message}</Alert>
```

Comment:
nit: You could've pulled in the MUI Alert, which I think is slightly nicer.

Reply:
It doesn't work with useAlert, but we have a useMUIAlert in the same file.

Reply:
Yeah I was already making big changes and didn't want to rock the boat (plus it was 2am 🤣 )